pyupgrade.py

```diff
@@ -57,26 +108,29 @@ def _convert_tup(tup):
     return j.join(_convert_tup(tup) for tup in parsed)
 
 
-def _ast_to_offset(node):
+def _ast_to_offset(node):  # type: (Union[ast.expr, ast.stmt]) -> Offset
     return Offset(node.lineno, node.col_offset)
 
 
-def ast_parse(contents_text):
+def ast_parse(contents_text):  # type: (Text) -> ast.Module
     # intentionally ignore warnings, we might be fixing warning-ridden syntax
     with warnings.catch_warnings():
-        warnings.simplefilter('ignore')
-        return ast.parse(contents_text.encode('UTF-8'))
+        warnings.simplefilter('ignore')  # type: ignore
```

Comment:
weird, do the stubs really not have `simplefilter`? can you link a bug for each `# type: ignore`

Reply:
it was a str/unicode issue (py2 mypy)